packages/python/instrumentation-sdk/src/infra/adapters/price_watcher.py

```
@@ -0,0 +1,53 @@
+            self._observer.start()
+        except Exception:
+            try:
+                self._observer = PollingObserver()
+                self._observer.schedule(handler, path=dir_to_watch, recursive=False)
+                self._observer.start()
+            except Exception:
+                self._observer = None
+
+    def _load(self) -> List[Dict[str, Any]]:
+        try:
+            with open(self._config_path, "r") as f:
```

Ruff (0.15.13)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)